libraries/botbuilder-dialogs-adaptive/src/adaptiveBotComponent.ts

```diff
@@ -121,7 +121,14 @@ import {
     UrlEntityRecognizer,
 } from './recognizers';
 
+/**
+ *
+ */
 export class AdaptiveBotComponent extends BotComponent {
+    /**
+     * @param services A list of services.
+     * @param _configuration The configuration.
+     */
```

Comment:
Use description and params from DotNet, i also could be defined in the BotComponent class and use the `@inheritdoc`